Controllers/WebSocketController.cs

```diff
@@ -19,9 +22,77 @@ public WebSocketController(ClientsCache clientsClients, ILogger<WebSocketControl
 
         public override async Task ReceiveAsync(WebSocket socket, ReadOnlyMemory<byte> buffer)
         {
-            _logger.Log(LogLevel.Debug, $"Message received: {Encoding.UTF8.GetString(buffer.Span)}");
-            var message = Encoding.UTF8.GetString(buffer.Span.Slice(5));
-            await SendMessageAsync(socket, $"Pong {message}");
+            var message = Encoding.UTF8.GetString(buffer.Span);
```

Comment:
We are gonna have to work with bytes. No strings.

Reply:
We don't have to LogInformation everytime we switch between cases since debug is already handling that.